src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -396,4 +396,13 @@ bool CoreWorkerMemoryStore::Contains(const ObjectID &object_id, bool *in_plasma)
   return false;
 }
 
+uint64_t CoreWorkerMemoryStore::UsedMemory() {
+  absl::MutexLock lock(&mu_);
+  uint64_t used_memory = 0;
+  for (const auto &it : objects_) {
+    used_memory += it.second->GetSize();
```

Comment:
You might need to check for `it->second->IsInPlasmaError()` and not access `GetSize()` in that case, a bit similar to `CoreWorkerMemoryStore::Contains`.

Reply:
For context: These are objects that were directly received by the worker, but were large, so they got put into the shared memory object store. So probably they shouldn't count against the core worker's memory use.